metisp/pymetis/src/pymetis/dataitems/hci/ifuHci.py

```diff
@@ -0,0 +1,360 @@
+    _frame_level = cpl.ui.Frame.FrameLevel.FINAL
+    _oca_keywords = {'PRO.CATG', 'INS.OPTI3.NAME', 'INS.OPTI9.NAME', 'INS.OPTI10.NAME', 'DRS.FILTER'}
+
+    _schema = {
+        'PRIMARY': None,
+        'DET1.DATA': Image,
+    }
+
+class IfuSciContrastRadprof(CgrphSpecificMixin, ImageDataItem, abstract=True):
+
+    _name_template = r'IFU_RAVC_SCI_CONTRAST_RADPROF'
+    _title_template = r"IFU RAVC sci contrast radprof"
+    _description_template = "" 
+    _frame_group = cpl.ui.Frame.FrameGroup.PRODUCT
+    _frame_level = cpl.ui.Frame.FrameLevel.FINAL
+    _oca_keywords = {'PRO.CATG', 'INS.OPTI3.NAME', 'INS.OPTI9.NAME', 'INS.OPTI10.NAME', 'DRS.FILTER'}
+
+    _schema = {
+        'PRIMARY': None,
+        'DET1.DATA': Table,
+    }
+
+class IfuSciContrastAdi(CgrphSpecificMixin, ImageDataItem, abstract=True):
+
+    _name_template = r'IFU_RAVC_SCI_CONTRAST_ADI'
+    _title_template = r"IFU RAVC sci contrast adi"
+    _description_template = "" 
+    _frame_group = cpl.ui.Frame.FrameGroup.PRODUCT
+    _frame_level = cpl.ui.Frame.FrameLevel.FINAL
+    _oca_keywords = {'PRO.CATG', 'INS.OPTI3.NAME', 'INS.OPTI9.NAME', 'INS.OPTI10.NAME', 'DRS.FILTER'}
+
+    _schema = {
+        'PRIMARY': None,
+        'DET1.DATA': Table,
+    }
+
+class IfuSciThroughput(CgrphSpecificMixin, ImageDataItem, abstract=True):
+
+    _name_template = r'IFU_RAVC_SCI_THROUGHPUT'
+    _title_template = r"IFU RAVC sci throughput"
+    _description_template = "" 
+    _frame_group = cpl.ui.Frame.FrameGroup.PRODUCT
+    _frame_level = cpl.ui.Frame.FrameLevel.FINAL
+    _oca_keywords = {'PRO.CATG', 'INS.OPTI3.NAME', 'INS.OPTI9.NAME', 'INS.OPTI10.NAME', 'DRS.FILTER'}
+
+    _schema = {
+        'PRIMARY': None,
+        'DET1.DATA': Table,
+    }
+
+class IfuSciCoverage(CgrphSpecificMixin, ImageDataItem, abstract=True):
+
+    _name_template = r'IFU_RAVC_SCI_COVERAGE'
+    _title_template = r"IFU RAVC sci coverage"
+    _description_template = "" 
+    _frame_group = cpl.ui.Frame.FrameGroup.PRODUCT
+    _frame_level = cpl.ui.Frame.FrameLevel.FINAL
+    _oca_keywords = {'PRO.CATG', 'INS.OPTI3.NAME', 'INS.OPTI9.NAME', 'INS.OPTI10.NAME', 'DRS.FILTER'}
+
+    _schema = {
+        'PRIMARY': None,
+        'DET1.DATA': Image,
+    }
+
+class IfuSciSnr(CgrphSpecificMixin, ImageDataItem, abstract=True):
+
+    _name_template = r'IFU_RAVC_SCI_SNR'
+    _title_template = r"IFU RAVC sci snr"
+    _description_template = "" 
+    _frame_group = cpl.ui.Frame.FrameGroup.PRODUCT
+    _frame_level = cpl.ui.Frame.FrameLevel.FINAL
+    _oca_keywords = {'PRO.CATG', 'INS.OPTI3.NAME', 'INS.OPTI9.NAME', 'INS.OPTI10.NAME', 'DRS.FILTER'}
+
+    _schema = {
+        'PRIMARY': None,
+        'DET1.DATA': Image,
+    }
+
+# IFU data types for metis_ifu_adi_cgrph
+
+class IfuRavcSciThroughput(CgrphAppMixin, IfuSciThroughput):
+    pass
+
+class IfuCvcSciThroughput(CgrphAppMixin, IfuSciThroughput):
+    pass
+
+class IfuAppSciThroughput(CgrphAppMixin, IfuSciThroughput):
+    pass
```

Comment:
These Mixin's are all for `App`, that doesn't seem right. Maybe
```suggestion
class IfuRavcSciThroughput(CgrphRavcMixin, IfuSciThroughput):
    pass

class IfuCvcSciThroughput(CgrphCvcMixin, IfuSciThroughput):
    pass

class IfuAppSciThroughput(CgrphAppMixin, IfuSciThroughput):
    pass
```